VMs/metrics/3_complete_baseimage.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# Copyright (c) 2018 Intel Corporation
+# 
+# SPDX-License-Identifier: Apache-2.0
+
+source global_env.sh
+
+die() {
+	echo "Fatal: $*" >&2
+	exit 1
+}
+
+error() {
+	echo "Error: $*" >&2
+}
+
+message() {
+	echo "Message: $*"
+}
+
+if [ $# != 1 ]; then
+	error "Missing first argument"
+	message "Usage: $0 distro_config_dirname"
+	die
+fi
+
+CONFIG_DIR=$1
+
+if [ ! -f ${CONFIG_DIR}/${ENV_FILE} ]; then
+	die "Env file ${CONFIG_DIR}/${ENV_FILE} missing"
+fi
+
+source ${CONFIG_DIR}/${ENV_FILE}
+
+# We could be smarter and check if it is already running...
```

Comment:
This looks like a FIXME? If so, you could turn it into an issue and reference here.

Reply:
I'll make it a FIXME. There are a bunch of extra smarts that could be added to the scripts, but prudency to get it working and out took priority